make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -184,3 +184,43 @@ CREATE TABLE audit_log
  username       varchar(255) NOT NULL,
  op_time        timestamp default CURRENT_TIMESTAMP
 );
+
+/*migrate access log to audit log*/
+/*TODO drop table access_log?*/
+DO $$
+DECLARE
+    access RECORD;
+BEGIN
+    FOR access IN SELECT * FROM access_log
+    LOOP
+      /*insert project create and delete*/
+      IF access.operation = 'create' OR access.operation = 'delete' THEN
+          INSERT INTO audit_log (project_id, operation, resource_type, resource, username, op_time) VALUES (access.project_id, access.operation, 'project', access.repo_name, access.username, access.op_time);
+      END IF;
+      /*insert push image -> create artifact*/
+      IF access.operation = 'push' THEN
+          INSERT INTO audit_log (project_id, operation, resource_type, resource, username, op_time) VALUES (access.project_id, 'create', 'artifact', CONCAT(access.repo_name,'/',access.repo_tag), access.username, access.op_time);
```

Comment:
Let's use `push` as the operation as the commands are `push` and the operation times are also `push time`?

Reply:
in the new OCI framework, do we have `push` action? Isn't it create an artifact?